wordle-web/src/components/NavBar.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <v-toolbar :elevation="4">
+    <template v-slot:prepend>
+      <v-btn icon to="/">
+        <v-icon color="success" size="x-large">mdi-earth</v-icon>
+      </v-btn>
+    </template>
+
+    <v-app-bar-title @click="$router.push('/')" class="font-weight-bold">
+      Word War 1
+    </v-app-bar-title>
+
+    <template v-slot:append>
+      <v-app-bar-nav-icon
+        size="x-large"
+        color="success"
+        @click="drawer = !drawer"
+      ></v-app-bar-nav-icon>
+    </template>
+
+    <v-navigation-drawer app v-model="drawer" rail rail-width="30px" location="right" temporary>
+      <template v-slot:prepend>
+        <v-btn
+          color="success"
+          size="medium"
+          class="ml-4 mt-1"
+          icon="mdi-close-circle-outline"
+          @click="drawer = !drawer"
+        ></v-btn>
+        <v-btn
+          color="success"
+          size="medium"
+          class="ml-3 mt-1"
+          icon="mdi-cog-outline"
+          @click="dialog = true"
+        ></v-btn>
+        <v-dialog v-model="dialog" width="40%">
+          <v-card>
+            <!-- -----------------------------FOR TESTING CUSTOM THEMES------------------------------>
```

Comment:
This color theme preview is super cool and useful for the dev. I personally was having the hardest time testing this when I couldn't see the colors.